tests/hpc/slurm_master_db.pm

```diff
@@ -137,7 +137,73 @@ sub test_06 {
 }
 
 sub test_07 {
-    ##TODO: Accounting check
+    my $name        = 'Slurm accounting';
+    my $description = 'Basic check for slurm accounting cmd';
+    my $result      = 0;
+    my %users       = (
+        'user_1' => 'Sebastian',
+        'user_2' => 'Egbert',
+        'user_3' => 'Christina',
+        'user_4' => 'Greg',
+    );
+
+    ##Add users TODO: surely this should be abstracted
+    script_run("useradd $users{user_1}");
+    script_run("useradd $users{user_2}");
+    script_run("useradd $users{user_3}");
+    script_run("useradd $users{user_4}");
+
+    script_run('sacctmgr -i add cluster linux');
+    my $cluster = script_output('sacctmgr -n -p list cluster');
+
+    if (index($cluster, 'linux') == -1) {
```

Comment:
the output of the cmd is

linux||0|0|1||||||||normal||

maybe you want to make sure string "linux" is there? You could simple do something like:

    if ( $cluster !=~ m/linux/ ) {

Reply:
sure, might be indeed more easy to read